web/tests/stream_data.test.cjs

```diff
@@ -322,10 +443,81 @@ test("basics", () => {
                 name: "social",
                 stream_id: 2,
                 subscribed: true,
+                can_add_subscribers_group: admins_group.id,
+                can_administer_channel_group: admins_group.id,
+                can_subscribe_group: admins_group.id,
             },
             unique_id: 2,
         },
     ]);
+
+    social.invite_only = true;
+    social.can_subscribe_group = me_group.id;
+    social.subscribed = false;
+
+    stream_starting_with_25.invite_only = true;
+    stream_starting_with_25.can_subscribe_group = nobody_group.id;
+    stream_starting_with_25.subscribed = false;
```

Comment:
I think `subscribed` field is already set as `false` above when defining the stream object, so this might not be needed.